bitbot/reminder.go

```diff
@@ -0,0 +1,259 @@
+package bitbot
+
+import (
+	"fmt"
+	"github.com/whyrusleeping/hellabot"
```

Comment:
Move 3rd party packages to a block below stdlib packages to make them easier to visually distinguish